DependencyInjection/CoopTilleulsAclSonataAdminExtensionExtension.php

```diff
@@ -33,5 +33,17 @@ public function load(array $configs, ContainerBuilder $container)
 
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
+
+        $container
+            ->getDefinition('coop_tilleuls_acl_sonata_admin_extension.acl.extension')
+            ->replaceArgument(
+                0,
+                new Reference(
```

Comment:
> ? I can't see it.

@meyerbaptiste sorry....problems with git, now It's here: https://github.com/vedro-compota/CoopTilleulsAclSonataAdminExtensionBundle/blob/patch-2/DependencyInjection/CoopTilleulsAclSonataAdminExtensionExtension.php#L17